cli/src/deploy/mod.rs

```diff
@@ -170,25 +171,42 @@ fn get_url() -> &'static str {
 }
 
 pub async fn get_config() -> HashMap<String, Vec<Config>> {
-  let anycloud_config = get_deploy_config().await;
-  let cred_configs = get_credentials().await;
+  let anycloud_prof = get_deploy_profile().await;
+  let cred_profs = get_cred_profiles().await;
   let mut all_configs = HashMap::new();
-  for (deploy_id, deploy_configs) in anycloud_config.into_iter() {
+  for (deploy_profile_name, deploy_profiles) in anycloud_prof.into_iter() {
     let mut configs = Vec::new();
-    for deploy_config in deploy_configs {
-      match cred_configs.get(&deploy_config.credentials) {
+    for profile in deploy_profiles {
+      let cred_prof_name = match profile.credentialProfile {
+        None => {
+          if cred_profs.len() != 1 {
+            let err = format!(
+              "No credential profile specified in deploy config {} when more than one \
+              credential profile exists in {}.",
+              deploy_profile_name, CREDENTIALS_FILE
+            );
+            eprintln!("{}", err);
+            error!("{}", err);
```

Comment:
Do you need both `eprintln!` and `error!` here? I thought `error!` would also print the message?

Reply:
so `eprintln` displays the message to the user and `error!` logs it for us, but the semantics of this change slightly with https://github.com/alantech/anycloud/pull/41 where `error!` goes to the `clientError` endpoint

Reply:
We could create a utility function that calls both when the messages are the same

Reply:
While having two backends for actual logging is weird, having one backend to a logging service and one backend to stderr makes total sense to me. I'd prefer if `error!` just splatted the message across whatever backends were connected, with `eprintln!` being a simple one that we have added to the list of backends to work with.

Reply:
Not a blocker for this PR, though.